src/main/java/core/basesyntax/FileWriter.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+import java.nio.file.Path;
+
+public interface FileWriter {
+    void write(String report, String file );
```

Comment:
The method signature for 'write' in the FileWriter interface should include a more descriptive parameter name for the file path. Using 'fileName' or 'filePath' instead of just 'file' would make the code more readable and self-documenting.